.github/workflows/ci.yml

```diff
@@ -91,14 +91,17 @@ jobs:
       shell: bash
       run: cargo $CMD build --verbose --target ${{ matrix.platform.target }} $OPTIONS --features $FEATURES
 
+    - name: Pin versions of dev-deps
+      if: matrix.rust_version == '1.65.0'
+      run: bash tools/fix_msrv.sh
```

Comment:
Can't we just call `cargo update -p half --precise 2.2.1`?

I would *prefer* not to use a bash script because the lack of cross-platform support kinda defeats it's purpose compared to just putting it directly into the CI script.

Reply:
I didn't know about this command, thanks!